test/README.md

```diff
@@ -0,0 +1,63 @@
+
+## Development ##
+Detailed information on developing in the [webrtc](https://github.com/webrtc) GitHub repo can be found in the [WebRTC GitHub repo developer's guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit?pli=1#heading=h.e3366rrgmkdk).
+
+This guide assumes you are running a Debian based Linux distribution (travis-multirunner currently fetches .deb browser packages).
+
+#### Clone the repo in desired folder
+```bash
+git clone https://github.com/webrtc/samples.git
+```
+
+#### Install npm dependencies
+```bash
+sudo npm install
+```
+
+#### Run tests
+Runs grunt and tests in test/tests.js.
+```bash
+npm test
+```
+
+#### Add tests
+test/tests.js is used as an index for the tests, tests should be added here using `require()`.
+The tests themselves should be placed in the same js folder as main.js: e.g.`src/content/getusermedia/gum/js/test.js`.
+
+The tests should be written using Testling for test validation (using Tape script language) and Selenium WebDriver is used to control and drive the test in the browser. 
+
+Use existing tests as guide on how to write tests and also look at the [Testling guide](https://ci.testling.com/guide/tape) and [Selenium WebDriver](http://www.seleniumhq.org/docs/03_webdriver.jsp) (make sure to select javascript as language preference.) for more information.
+
+Global Selenium WebDriver settings can be found in `test/selenium-lib.js`, if your test require some specific settings not covered in selenium-lib.js, add your own to the test and do not import the selenium-lib.js file into the test, only do this if it's REALLY necessary.
+
+Once your test is ready, create a pull request and see how it runs on travis-multirunner.
+
+#### Change browser and channel/version for testing
+Chrome stable is currently installed as the default browser for the tests.
+
+Currently Chrome and Firefox are supported[*](#-experimental-browser-support), check [travis-multirunner](https://github.com/DamonOehlman/travis-multirunner/blob/master/) repo for updates around this.
+Firefox channels supported are stable, beta and nightly.
+Chrome channels supported on Linux are stable, beta and unstable.
+
+To select a different browser and/or channel version, change environment variables BROWSER and BVER, then you can rerun the tests with the new browser.
+```bash
+export BROWSER=firefox BVER=nightly
+```
+
+Alternatively you can also do it without changing environment variables.
+```bash
+BROWSER=firefox BVER=nightly npm test
+```
+
+###* Experimental browser support###
+You can run the tests in any currently installed browser locally that is supported by Selenium WebDriver but you have to bypass travis-multirunner. Also it only makes sense to use a WebRTC supported browser.
```

Comment:
Not sure, we do not include browserify as a dependency in this repo, but you can do `BROWSER=firefox node src/content/getusermedia/gum/js/test.js to run specific tests in the browser specified.

Adding this to the instructions.